tests/unit/shader_object.cpp

```diff
@@ -1339,16 +1339,8 @@ TEST_F(NegativeShaderObject, InvalidExclusiveScissorCount) {
     AddRequiredFeature(vkt::Feature::multiViewport);
     AddRequiredFeature(vkt::Feature::exclusiveScissor);
     RETURN_IF_SKIP(InitBasicShaderObject());
-    uint32_t count;
-    vk::EnumerateDeviceExtensionProperties(m_device->Physical(), nullptr, &count, nullptr);
-    std::vector<VkExtensionProperties> properties(count);
-    vk::EnumerateDeviceExtensionProperties(m_device->Physical(), nullptr, &count, properties.data());
-    for (const auto& p : properties) {
-        if (std::strcmp(p.extensionName, VK_NV_SCISSOR_EXCLUSIVE_EXTENSION_NAME) == 0) {
-            if (p.specVersion < 2) {
-                GTEST_SKIP() << "exclusiveScissor specVersion 2 not supported.";
-            }
-        }
+    if (!DeviceExtensionSupported(VK_EXT_DISCARD_RECTANGLES_EXTENSION_NAME, 2)) {
```

Comment:
`VK_NV_SCISSOR_EXCLUSIVE_EXTENSION_NAME` is being replaced with `VK_EXT_DISCARD_RECTANGLES_EXTENSION_NAME`, is this intended or copy paste error?

Reply:
100% copy-and-paste error, will just fix this to be

```suggestion
    if (!DeviceExtensionSupported(VK_NV_SCISSOR_EXCLUSIVE_EXTENSION_NAME, 2)) {
```

(See I have one below as well)

Reply:
fixed